std/meta.d

```diff
@@ -954,6 +954,81 @@ unittest
     }
 }
 
+/**
+ * Sorts a AliasSeq using $(D C).
+ *
+ * Parameters:
+ *     $(D C) = A template that returns a $(D bool) (if its first argument
```

Comment:
Explicit formatting of the parameter names (left side of the `=` sign) should not be necessary; if we want to customize their formatting, we should do it project-wide by overriding a DDoc macro.